app/Controllers/Transactions.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace App\Controllers;
+
+use App\Models\PaymentModel;
+
+
+
+
+class Transactions extends BaseController
+{
+//Function to view tenant's payment history
+	public function tenantHistory($tenantID)
+	{
+		$db = db_connect();
+		$model = new PaymentModel($db);
+
+		$tenant = $model->isTenant($tenantID);
+		
+		if ($tenant == true) {
+			
+	
+			$result = $model->getTenantHistory($tenantID);
+
+
+			$data = ['data' => $result];
+		
+
+		}
+		else{
+			$data = ['data' => 'User not a Tenant'];
+		}
+
+
+		return json_encode($data);
+
+	}
+
+//Function to make payments
+	public function makePayment($tenantID)
+	{
+
+		$db = db_connect();
+		$model = new PaymentModel($db);
+
+
+
```

Comment:
A Tenant can have multiple properties...this won't work. The property beign paid for should be received from the front end as a POST request